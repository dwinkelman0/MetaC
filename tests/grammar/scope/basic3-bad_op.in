{
    int x = 0;
    const char *str, int thing;
    struct Result_t res = func(x, str);
}