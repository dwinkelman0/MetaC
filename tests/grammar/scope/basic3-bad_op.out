{
    int x = 0;
    struct Result_t res = func(x, str);
}